odoo/models.py

```diff
@@ -3918,9 +3918,9 @@ def _create(self, data_list):
         quote = '"{}"'.format
 
         # insert rows
-        ids = []                        # ids of created records
-        other_fields = OrderedSet()            # non-column fields
-        translated_fields = OrderedSet()       # translated fields
+        ids = []                                # ids of created records
+        other_fields = OrderedSet()             # non-column fields
+        translated_fields = OrderedSet()        # translated fields
```

Comment:
Or we unalign everything, possibly even rename the first to locals & remove the comments entirely :D